shared/utils/dateFilterUtils.ts

```diff
@@ -0,0 +1,75 @@
+import { DateTime } from 'luxon'
+
+import { DateModel, EventModel, LocationModel } from '../api'
+import { MAX_DATE_RECURRENCES } from '../constants'
+
+export const EventModalDummyData = {
+  content: '<h1>Event Content</h1>',
+  thumbnail: null,
+  location: new LocationModel({
+    id: 1,
+    name: 'Test Location',
+    address: 'Test Street',
+    town: 'Test Town',
+    postcode: '12345',
+    country: 'Test Country',
+    latitude: null,
+    longitude: null,
+  }),
+  excerpt: 'This is a test event.',
+  availableLanguages: {},
+  lastUpdate: DateTime.fromISO('2024-11-07T00:00:00.000'),
+  featuredImage: null,
+  poiPath: '/test/location/path',
+}
```

Comment:
I used the same dummyDate in two places `dateFilterUtils.spec` and `filteringEventsLogic.spec` if I moved this to a test file it will say `Do not export from a test file` and if I removed the export I will duplicate the same object twice in two files.

Reply:
Ah, good point, you put it in a file that doesn't have the .spec part in the name. You can also have repetitions in test files, that happens all the time, so not a big issue.